src/decrypter.js

```diff
@@ -7,8 +7,15 @@
 
 import window from 'global/window';
 import AES from './aes';
-import AsyncStream from './async-stream';
 import {unpad} from 'pkcs7';
+import AsyncStream from './async-stream';
+import Promise from 'promiz';
+
+// Need to make the Promise polyfill global from a node-style import
+// because the webcrypto library isn't really a node-style library
+window.Promise = window.Promise || Promise;
+
+let webcrypto = require('webcrypto-shim-jon');
```

Comment:
This import is never used directly. Does it modify `window`? If so, does it need to be assigned to anything?